Userland/Libraries/LibWeb/HTML/HTMLTextAreaElement.cpp

```diff
@@ -285,7 +285,15 @@ void HTMLTextAreaElement::children_changed()
 void HTMLTextAreaElement::attribute_changed(FlyString const& name, Optional<String> const& value)
 {
     HTMLElement::attribute_changed(name, value);
-    if (name == HTML::AttributeNames::placeholder) {
+    if (name == HTML::AttributeNames::value) {
+        if (!m_dirty_value) {
+            if (value.has_value()) {
+                (void)set_value(*value);
```

Comment:
Why are these `(void)` casts needed? Doesn't this return void?

Reply:
They return a exception which we can ignore

Reply:
I don't think so?

```
void HTMLTextAreaElement::set_value(String const& value)
```